pkg/operator/render/options/generic.go

```diff
@@ -0,0 +1,127 @@
+package options
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"text/template"
+
+	"github.com/ghodss/yaml"
+	"github.com/openshift/library-go/pkg/assets"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+	"github.com/spf13/pflag"
+)
+
+// GenericOptions contains the generic render command options.
+type GenericOptions struct {
+	ConfigOverrideFiles []string
+	ConfigOutputFile    string
+
+	TemplatesDir   string
+	AssetInputDir  string
+	AssetOutputDir string
+}
+
+// LoadAsset loads asset files.
+type LoadAsset func(name string) []byte
+
+// AddFlags adds the generic flags to the flagset.
+func (o *GenericOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.AssetOutputDir, "asset-output-dir", "", "Output path for rendered manifests.")
+	fs.StringVar(&o.AssetInputDir, "asset-input-dir", "", "A path to directory with certificates and secrets.")
+	fs.StringVar(&o.TemplatesDir, "templates-input-dir", "/usr/share/bootkube/manifests", "A path to a directory with manifest templates.")
```

Comment:
don't do defaults like this.  Default values based on current values and have a `NewOptions` method.